.gitignore

```diff
@@ -32,5 +32,8 @@ deploy/settings.yml
 deploy/google/app.yaml
 !deploy/google/app.yaml.example
 
+# Virtual environment
+venv/
```

Comment:
Please add these changes to your local .gitignore file. They shouldn't be in .gitignore.